tests/run-pass/close-stderr.rs

```diff
@@ -0,0 +1,16 @@
+// compile-flags: -Zmiri-disable-isolation
```

Comment:
This will need "ignore-windows" as well.

Reply:
Added this.